EcoSoapBank/Pickups/PickupCoordinator.swift

```diff
@@ -116,7 +116,7 @@ extension PickupCoordinator {
             style: .default,
             handler: { _ in
                 UIApplication.shared.open(
-                    pickupResult.labelURL,
+                    pickupResult.labelURL!,
```

Comment:
Yeah, don't leave this in here too long. This alert should not give option to view/print if there is no labelURL.

Reply:
That makes sense.